src/components/MultipleBeneficiariesAmount.tsx

```
@@ -0,0 +1,294 @@
+
+	const checkOnGenralIndexBeneficiaries = async () => {
+		if (beneficiaries?.length == 1 || !beneficiaries?.length) return { genralIndex: beneficiaries[0]?.genralIndex, isSame: true };
+		const firstGenralIndex = beneficiaries[0]?.genralIndex;
+		return {
+			genralIndex: beneficiaries?.find((beneficiary) => beneficiary?.genralIndex)?.genralIndex,
+			isSame: beneficiaries.every((item) => item.genralIndex === firstGenralIndex)
+		};
+	};
+
+	const fetchUSDValue = async () => {
+		if (!proposalCreatedAt || dayjs(proposalCreatedAt).isSame(dayjs()) || (postId && isNaN(postId))) return;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)